test/rocprim/test_device_merge_sort.cpp

```diff
@@ -22,6 +22,10 @@
 
 #include "common_test_header.hpp"
 
+#ifdef WIN32
+#include <numeric>
```

Comment:
We should put this to ```common_test_header.hpp``` and you can reduce the extra lines.

Reply:
Done. Fixed in upcoming update to this PR.